docs/concepts/v2/concepts/prebuilt.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Prebuilt Room Links
+nav: 1.9
+---
+Embed or share room links without building UI
```

Comment:
can we add a few screenshots here? as a new user, i would want to see what the UI looks like